setup.py

```diff
@@ -48,7 +48,7 @@
     # Going to update in a proper commit
     "cairosvg>=2.5.2",
     "pycurl==7.*,>=7.43.0",
-    "py3exiv2==0.*,>=0.7.1,!=0.7.2,!=0.8.0,!=0.9.3",
+    "py3exiv2>=0.*,<0.7.2",
```

Comment:
Either we keep it like this **and** have some other mechanism to try new releases of `py3exiv2` or we add the offending release to the "black list".